scripts/extract-l10n.sh

```diff
@@ -8,10 +8,10 @@
 #   - Update your code
 #   - Extract new strings and push to the .po files
 #
-# If you're running the script manually please make sure to expose
+# For the fully automated experience make sure to expose
 # the following variables to the environment:
 #   - GITHUB_TOKEN (to the github token of addons-robot,
-#                   talk to tofumatt or cgrebs)
+#                   talk to @diox or @muffinresearch)
 #   - TRAVIS_REPO_SLUG="mozilla/addons-server"
 #   - TRAVIS_BRANCH="master"
 
```

Comment:
In `init_environment` it does `git checkout master` which depends on you having an up-to-date local master.  Can we just `git checkout upstream/master` instead?   (Even better if we can always checkout from the repo in `TRAVIS_REPO_SLUG`)

Reply:
mmm in the case of travis, `upstream` doesn't exist as the remote... I'm going to try to steal some logic from https://github.com/mozilla/addons-frontend/blob/master/bin/run-l10n-extraction for this, pulling from the remote directly.